core/src/Kokkos_Core.hpp

```diff
@@ -258,6 +258,9 @@ class ScopeGuard {
 
 }  // namespace Kokkos
 
+#if defined(KOKKOS_ENABLE_VIEW_HOOKS)
+#include <impl/Kokkos_ViewHookDeepCopy.hpp>
+#endif
```

Comment:
I'd rather include this unconditionally and move the `#if` to `Kokkos_ViewHookDeepCopy.hpp`. I actually don't see a reason to have this disabled from the build system, though.

Reply:
option removed 60b42a7d87599a9d2c2bfe913cb811b38a3b6f9f